### ISSUE TITLE
Nifskope 2.0.x and WIndows 10x64

### ISSUE THREAD

**Issue by ThisssIssQwe** [niftools/nifskope#130]
Hello everyone. I have a problem. Nifskope 2.0.x does not start at Windows 10x64 Home v.1709 build 16299.192. When I trying to run NidSkope.exe, window of program does not appearing. At system log (application) I can find this message "The name of the failing application: NifSkope.exe, version: 0.0.0.0, timestamp: 0x58207f38 The name of the failing module: Qt5Gui.dll, version: 5.7.0.0, time stamp: 0x575a6ad5 Exception code: 0xc0000005". I have tryed to use 2.0.6 and 2.0.7 version of program. But version 1.2.0 works correctly. What can you advise?

**Comment by yoosYyd** [comment]
I have same problem.

**Comment by iansebryk** [comment]
yep.  total deer-in-the-headlights going on.  launches, then freezes solid.  same situation as OP, except 1.2.0 alpha 2 also fails.  any advice?

**Comment by ThisssIssQwe** [comment]
One clever man says that turning HDR off in Windows settings halped him. At my computer this option desabled and Nifskope does not start. Perheps this information could help you.
